lte/gateway/python/magma/mobilityd/ip_allocator_static.py

```diff
@@ -23,21 +23,25 @@
 
 from magma.mobilityd.ip_descriptor import IPDesc, IPState, IPType
 from magma.mobilityd.ip_allocator_base import IPAllocator
-from magma.mobilityd.subscriberdb_client import SubscriberDbClient
+from magma.mobilityd.subscriberdb_client import SubscriberDbClient, StaticIPInfo
+from magma.mobilityd.uplink_gw import UplinkGatewayInfo
+import logging
 
 DEFAULT_IP_RECYCLE_INTERVAL = 15
 
 
 class IPAllocatorStaticWrapper(IPAllocator):
 
-    def __init__(self, subscriberdb_rpc_stub, ip_allocator: IPAllocator):
+    def __init__(self, subscriberdb_rpc_stub, ip_allocator: IPAllocator,
+                 gw_info: UplinkGatewayInfo):
         """ Initializes a static IP allocator
             This is wrapper around other configured Ip allocator. If subscriber
             does have static IP, it uses underlying IP allocator to allocate IP
             for the subscriber.
         """
         self._subscriber_client = SubscriberDbClient(subscriberdb_rpc_stub)
         self._ip_allocator = ip_allocator
+        self._gw_info = gw_info
 
     def add_ip_block(self, ipblock: ip_network):
```

Comment:
not related to your changes but let's add typing for the return type `Optional[IPDesc]`